appalerts/templates/appalerts/alert_list.html

```diff
@@ -83,15 +107,17 @@ <h2><i class="iconify" data-icon="mdi:alert-octagram"></i> App Alerts</h2>
                 <i class="iconify" data-icon="mdi:arrow-right-top-bold"></i>
               </a>
             {% endif %}
-            <a class="btn btn-{{ a.level | lower }}
+            {% if not dismissed %}
```

Comment:
The "Follow and dismiss" button should also be hidden in the dismissed alert view.